lib/taskana-core/src/main/java/pro/taskana/impl/TaskQueryImpl.java

```diff
@@ -420,7 +421,7 @@ public TaskQuery stateIn(TaskState... states) {
     @Override
     public TaskQuery stateNotIn(TaskState... states) {
         // No benefit in introducing a new variable
-        List<TaskState> stateIn = Arrays.asList(TaskState.values());
+        List<TaskState> stateIn = new LinkedList<TaskState>(Arrays.asList(TaskState.values()));
```

Comment:
it had an error because of arrays.asList(list) had a fixed size so, when you try to remove one element this throw an exception. 
You can see more information on this link:
https://stackoverflow.com/questions/2965747/why-do-i-get-an-unsupportedoperationexception-when-trying-to-remove-an-element-f

Reply:
Exactly, it doesn't allow modify the size of the list